.travis.yml

```diff
@@ -61,7 +55,7 @@ script:
   - ./gradlew --version
   - ./gradlew clean
   - if [ "$TRAVIS_JDK_VERSION" == "oraclejdk9" ]; then ./gradlew check; fi
-  - ./gradlew cobertura coveralls
+  - ./gradlew -Djdk.tls.client.protocols="TLSv1.2" cobertura coveralls
```

Comment:
I had to add this option, otherwise uploading the report to coveralls.io fails with an SSL error with Java 11. My best guess is that Java 11 tries to use TLS 1.3 and doesn't fall back to 1.2.

Reply:
Wow, Java 11 is just too advanced!